Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -523,7 +557,7 @@ class SyncedTabsButton: UIButton {
     }()
 
     lazy var chevron: ChevronView = {
-        let chevron = ChevronView(direction: .Right)
+        let chevron = ChevronView(direction: UIApplication.sharedApplication().userInterfaceLayoutDirection == .LeftToRight ? .Right : .Left)
```

Comment:
Rather than requiring this check everywhere we want to use a `ChevronView`, I'd move this interface direction check inside of `ChevronView` directly. Then, everywhere we pass `Left`, `ChevronView` would internally draw a right arrow for RTL (and vice versa).